dev-python/pyside/metadata.xml

```diff
@@ -6,18 +6,27 @@
 		<name>Gentoo Qt Project</name>
 	</maintainer>
 	<use>
-		<flag name="X">Build QtGui and QtTest modules</flag>
-		<flag name="declarative">Build QtDeclarative module</flag>
-		<flag name="designer">Build QtDesigner and QtUiTools modules</flag>
-		<flag name="help">Build QtHelp module</flag>
-		<flag name="multimedia">Build QtMultimedia module</flag>
-		<flag name="opengl">Build QtOpenGL module</flag>
-		<flag name="script">Build QtScript module</flag>
-		<flag name="scripttools">Build QtScriptTools module</flag>
-		<flag name="sql">Build QtSql module</flag>
-		<flag name="svg">Build QtSvg module</flag>
-		<flag name="webkit">Build QtWebKit module</flag>
-		<flag name="xmlpatterns">Build QtXmlPatterns module</flag>
+		<flag name="concurrent">Build Qt5Concurrent module</flag>
```

Comment:
*Woopsie.* Thanks for the subtle catch. Never did like the number "5," anyway. It shall go away.